services/converters.go

```diff
@@ -220,9 +220,10 @@ func ToAPIAgent(q *reform.Querier, agent *models.Agent) (inventorypb.Agent, erro
 	switch agent.AgentType {
 	case models.PMMAgentType:
 		return &inventorypb.PMMAgent{
-			AgentId:      agent.AgentID,
-			RunsOnNodeId: pointer.GetString(agent.RunsOnNodeID),
-			CustomLabels: labels,
+			AgentId:         agent.AgentID,
+			RunsOnNodeId:    pointer.GetString(agent.RunsOnNodeID),
+			CustomLabels:    labels,
+			ProcessExecPath: *agent.ProcessExecPath,
```

Comment:
please use pointer lib to get value instead of dereference in all places

Reply:
fixed